lib/database/bootstrapper_runtime/Dockerfile

```diff
@@ -4,8 +4,8 @@ FROM --platform=linux/amd64 public.ecr.aws/lambda/python:${PYTHON_VERSION}
 RUN echo "PYTHON_VERSION: ${PYTHON_VERSION}"
 
 WORKDIR /tmp
-
-RUN pip install httpx psycopg[binary,pool] pypgstac -t /asset
+ARG PGSTAC_VERSION
+RUN pip install httpx psycopg[binary,pool] pypgstac==${PGSTAC_VERSION} -t /asset
```

Comment:
I added this because it is possible that you could try to upgrade to a new version of `pypgstac` via migration but if the bootstrapper runtime is running an older version you can't migrate up to a newer version since it is unaware of the new migration path.

Reply:
👍